src/dreyfus_index_updater.erl

```diff
@@ -64,7 +64,7 @@ update(IndexPid, Index) ->
         couch_db:close(Db)
     end.
 
-load_docs(FDI, {I, IndexPid, Db, Proc, Total, LastCommitTime}=Acc) ->
+load_docs(FDI, {I, IndexPid, Db, Proc, Total, LastCommitTime, DbName}=Acc) ->
```

Comment:
do we need to pass in DbName here? would `couch_db:name(Db)` return what's needed?

Reply:
I tried that and it didn't work. But the is db partitioned work in mem3 is likely to change. So I'll update the branch once it does.